src/Constraint/AbstractConstraint.php

```diff
@@ -19,49 +19,9 @@ abstract class AbstractConstraint
      * @param string $name The constraint name.
      * @param string[] $columnNames The list of column names the constraint belongs to.
      */
-    public function __construct(private string $name = '', private array $columnNames = [])
-    {
-    }
-
-    /**
-     * @return string[] The list of column names the constraint belongs to.
-     *
-     * @psalm-immutable
-     */
-    public function getColumnNames(): array
-    {
-        return $this->columnNames;
-    }
-
-    /**
-     * @return string The constraint name.
-     *
-     * @psalm-immutable
-     */
-    public function getName(): string
-    {
-        return $this->name;
-    }
-
-    /**
-     * Set the list of column names the constraint belongs to.
-     *
-     * @param string[] $value The list of column names the constraint belongs to.
-     */
-    public function columnNames(array $value): static
-    {
-        $this->columnNames = $value;
-        return $this;
-    }
-
-    /**
-     * Set the constraint name.
-     *
-     * @param string $value The constraint name.
-     */
-    public function name(string $value): static
-    {
-        $this->name = $value;
-        return $this;
+    public function __construct(
```

Comment:
Seems, `AbstractConstraint` is never used: https://github.com/search?q=org%3Ayiisoft+abstractconstraint&type=code

May be remove this class?

Reply:
Good idea, I'll remove it